example/windows/window_configuration.cpp

```diff
@@ -14,7 +14,7 @@
 
 #include "window_configuration.h"
 
-const wchar_t *kFlutterWindowTitle = L"Flutter Desktop Example";
+const wchar_t *kFlutterWindowTitle = L"Flutter Desktop Example";  //Change 'Flutter Desktop Example' to the Your Application Name 

```

Comment:
This is already explained in a README with other information about customization.